spoon/form/form.php

```diff
@@ -1140,10 +1140,17 @@ public function validate()
 		$errors = [];
 
 		// if we use tokens, we validate them here
-		if($this->getUseToken())
+		if($this->getUseToken() && $this->isSubmitted())
 		{
-			if ($this->getMethod() === 'get' && !isset($_GET['form_token'])
-				|| $this->getMethod() === 'post' && !isset($_POST['form_token'])) {
+			$submittedToken = '';
+			if ($this->getMethod() === 'get' && !isset($_GET['form_token']) {
```

Comment:
i think there is still something wrong here. You are checking if the method is get and if there isn't a form token in $_GET. If that is the case you try to get the not existing token out of $_GET. 
The same happens with $_POST

Reply:
You also forgot to close a ) here and with the post one

Reply:
Fixed in 68eb811cc8a2c26686d40f9c317b34534d1c2293